### ISSUE TITLE
Is it possible to use Scanner to load templates from multiple mountpoints?

### ISSUE THREAD

**Issue by mrlacey** [dotnet/templating#4439]
I have a test scenario where I want to load templates from different places on disk but can't see a way to do this. Am I missing something obvious? Is this possible?

I can work around this by copying all the templates to a single location and loading them all from there but it creates an overhead for tidying up all the files afterward.

Background: we have templates that add StyleCop references and settings files that we want to use to verify consistency in the code generated from our other templates, but it isn't appropriate to include the StyleCop related files in the distributed templates. Keeping them separate within our project structure (rather than excluding the test-specific ones at runtime) helps us manage the project more easily.

**Comment by bekir-ozturk** [comment]
Hey @mrlacey,
 Thanks for the issue.
 Is calling the following method multiple times for each template directory not doing the job for you?
https://github.com/dotnet/templating/blob/329b2aa43d80398b25aa63028711a91bfff835ec/src/Microsoft.TemplateEngine.Edge/Settings/Scanner.cs#L44

**Comment by mrlacey** [comment]
> Hey @mrlacey, Thanks for the issue. Is calling the following method multiple times for each template directory not doing the job for you?
> 
> https://github.com/dotnet/templating/blob/329b2aa43d80398b25aa63028711a91bfff835ec/src/Microsoft.TemplateEngine.Edge/Settings/Scanner.cs#L44

Yes, that works. But, I saw that internally it was using a collection of mountpoints. 
I was wondering if there was a way to pass multiple mountpoints and have it combine multiple responses for me. 

I assume it's a fairly niche requirement and there are probably good arguments for me managing different possible errors and partial scan success, but I was wondering (hoping) if there was an opportunity for me to be lazy here.

**Comment by vlada-shubina** [comment]
@mrlacey 
We understand the concern, but we don't plan to adjust this API in near future. 
In our opinion, the current API doesn't have any limitations, just not convenient to work with in case of multiple mount points. 
The management of multiple mount points and error handling might be implemented on caller side.

Therefore, I'm closing it now, however we gladly take a PR if anyone wants to contribute and implement it.
